contrib/stack/topsStack/mergeBursts.py

```diff
@@ -390,8 +391,7 @@ def main(iargs=None):
         fileList.append([os.path.join(inps.dirname, 'IW{0}'.format(swath), namePattern[0] + '_%02d.%s'%(x,namePattern[1])) for x in range(minBurst, maxBurst+1)])
 
     mergedir = os.path.dirname(inps.outfile)
-    if not os.path.isdir(mergedir):
-        os.makedirs(mergedir)
+    pathlib.Path(mergedir).mkdir(parents=True, exist_ok=True)
```

Comment:
```suggestion
    os.makedirs(mergedir, exist_ok=True)
```
Then you don't need to use pathlib

Reply:
Good idea! That makes it much easier.

Reply:
Oh, and you can now remove the `import pathlib` too :)